src/test/groovy/unit/org/pih/warehouse/requisition/RequisitionTests.groovy

```diff
@@ -141,26 +147,38 @@ class RequisitionTests {
     */
 
      void testDateRequestedCanNeToday() {
-        mockForConstraintsTests(Requisition)
-        def requisition = new Requisition(dateRequested:new Date())
-        requisition.validate()
-        assertNull requisition.errors["dateRequested"]
+         given:
+         def requisition = new Requisition(dateRequested:new Date())
```

Comment:
```suggestion
         Requisition requisition = new Requisition(dateRequested: new Date())
```

Reply:
I agree. I didn't change this one because I wanted to show the minimum number of changes needed to refactor the test but since I'm changing that line anyways, yes I'll update it.

Reply:
actually, these defs are all over the file and so it'd feel weird to just refactor this one case and not go through and refactor all of the tests, many of which are unchanged, so I'm just going to leave it as is for now.

Reply:
I am fine with that :smile: